src/api/app/views/webui/project/subprojects.html.haml

```diff
@@ -6,34 +6,48 @@
 
 - if @parentprojects.present?
   %h3 Parent projects of #{@project}
-  %table.compact.stripe.no-footer#parentprojects_table
+  %table.compact.stripe.no-footer{:id => "parentprojects_table", :style => "table-layout:auto; width: 100%;word-wrap:break-word;"}
     %thead
       %tr
         %th Parent project
-        %th Description
+        %th Title
     %tbody
       - @parentprojects.each do |project|
         %tr
-          %td= link_to project.name, :action => 'show', :project => project.name
-          %td= project.title
+          %td{:style => "max-width: 20em;min-width: 20em;"}= link_to(word_break(project.name, 12), {:action => 'show', :project => project.name})
+          %td{:style => "max-width: 30em;min-width: 30em;"}= project.title
 
 %h3= @pagetitle
 - prjlen = @project.name.length
 - if @subprojects.present?
-  %table.compact.stripe.no-footer#subprojects_table
+  %table.compact.stripe.no-footer{:id => "subprojects_table", :style => "table-layout:auto; width: 100%;word-wrap:break-word;"}
     %thead
       %tr
         %th Subproject
-        %th Description
+        %th Title
     %tbody
       - @subprojects.each do |project|
         %tr
-          %td= link_to project.name.slice(prjlen + 1, project.name.length), :action => 'show', :project => project.name
-          %td= project.title
+          %td{:style => "max-width: 20em;min-width: 20em;"}= link_to(word_break(project.name.slice(prjlen + 1, project.name.length), 12), {:action => 'show', :project => project.name})
+          %td{:style => "max-width: 30em;min-width: 30em;"}= project.title
 - else
   %p
     %i This project has no subprojects
 
+- if @siblings.present?
+- parent_len = @parent.name.length
+  %h3 Sibling projects of #{@project}
+  %table.compact.stripe.no-footer{:id => "siblingprojects_table", :style => "table-layout:auto; width: 100%;word-wrap:break-word;"}
+    %thead
+      %tr
+        %th Sibling project
+        %th Title
+    %tbody
+      - @siblings.each do |project|
+        %tr
+          %td{:style => "max-width: 20em;min-width: 20em;"}= link_to project.name.slice(parent_len + 1, project.name.length), {:action => 'show', :project => project.name}
```

Comment:
A helper function for 
```
project.name.slice(parent_len + 1, project.name.length)
```
would be nice as it is really hard to understand what it is doing ...

Something like
```
project.parent_name
```
would be awesome. Or a view helper
```
remove_parent_name(project, parent)
```